wormhole/module.py

```diff
@@ -203,6 +223,7 @@ async def on_message(self, message: discord.Message):
                     for message_part in formatted_message_parts:
                         await target_channel.send(
                             message_part,
+                            files=copy.deepcopy(files_list),
```

Comment:
Please reset the file pointers for the files instead of doing a deep copy. Should be faster and cleaner.